### DISCUSSION TITLE
Change terminationGracePeriodSeconds from values.yaml

### DISCUSSION THREAD

**Discussion by AndreCbrera** [goharbor/harbor-helm#1758] [Q&A] [answered]
Morning !

I don't know it is possible but it would be amazing to be able to handle this parameter from values.yaml:

[database StatefulSet](https://github.com/goharbor/harbor-helm/blob/f547605c75badb6675810327cc4809b3ec4202a8/templates/database/database-ss.yaml#L44C7-L44C36)
[redis StatefulSet](https://github.com/goharbor/harbor-helm/blob/f547605c75badb6675810327cc4809b3ec4202a8/templates/redis/statefulset.yaml#L43)

trivy do not have this section but applies the **terminationGracePeriodSeconds: 120** by default

We have a kubernetes in On-Premise and right now we modified this section to apply a solution and change the  **terminationGracePeriodSeconds: 0**

Thanks

**Comment by zyyw** [comment, marked as answer]
Hi @AndreCbrera trivy doesn't have `terminationGracePeriodSeconds` section, it uses default value `30`.
[The default terminationGracePeriodSeconds setting is 30 seconds.](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/)
